examples/listeners.ssl.conf.example

```diff
@@ -0,0 +1,66 @@
+##--------------------------------------------------------------------
+## SSL Listener
+##
+## Add a SSL Listener
+##--------------------------------------------------------------------
+## Note: This is an example of how to configure this feature
+##       you should copy and paste the below data into the emqx.conf for working
+
+## The SSL listener also supports all the fields listed in listeners.tcp.conf.example
+## only the SSL-specific fields are shown here
+
+## Note: Modifying the 'sslname' to what you need
+listeners.ssl.sslname {
+    ## Whether to enable the listener
+    enable = true
+
+    ## Port or Address to listen on
+    bind = 8883 ## or with an IP e.g. "127.0.0.1:8883"
+
+    ## Trusted PEM format CA certificates bundle file
+    cacertfile = "data/certs/cacert.pem"
+
+    ## PEM format certificates chain file
+    certfile = "data/certs/cert.pem"
+
+    ## PEM format private key file
+    keyfile = "data/certs/key.pem"
+
+    ## Enable or disable peer verification
+    verify = verify_none  ## use verify_peer to enable
+
+    ## if `verify' is ebabled, whit true, the connection fails if the client does not have a certificate to send
+    fail_if_no_peer_cert = false
+
```

Comment:
It's wrong, should in ssl_options, you can get all config with default 
`emqx:get_config([listeners,ss,default]).